### ISSUE TITLE
Implement ticketing and payments API

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#3468]
Parent issue #3459
To be done after #3645

We need to be able to initiate purchases via API. The following are required.
- API to create order and get payment link
- API to set order purchase status after payment (would need encryption and verification of the data with the payment gateway server for validity before accepting)
- API to view current orders of a user (along with status)
- API to edit order details. (More info #2996)
- API to download ticket pdf of a ticket
- API to view tickets purchased by a user
- API to delete ticket (refund to be initiated based on admin settings) (more info #3036)

This is a parent issue. Sub-issues can be created.

**Comment by SaptakS** [comment]
This issue is already covered in the above mentioned issues. Closing.
